docs/apps/components/collapse.md

```diff
@@ -0,0 +1,117 @@
+7. **Accessibility considerations**: Optimize the collapse component for users with disabilities.
+
+## API
+
+### Methods
+
+| **Name**| **Description**|**Parameters**|
+|---------|----------------|--------------|
+|`this.addClassName`|Add style class to a form item|`(fields: String`&#124;`String[], className: String)`|
+|`this.removeClassName`|Remove style class from a form item|`(fields: String`&#124;`String[], className: String)`|
+|`this.getComponent`|Returns a component whose id has been passed as a parameter|`(component_ID: String) : Object`|
+|`this.hide`|Hides the field|`(fields: String`&#124;`String[])`|
+|`this.show`|Displays the field|`(fields: String`&#124;`String[])`|
+
+!!! Info
+    1. The show() and hide() methods can also be used to control the visibility of an alert in response to user input.
+    2. **Fields** refers to a **component ID** or a **list of component IDs**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. Before using **this.show(fields)**, make sure the component is **hidden**. This can be done using **this.hide(fields)** or by **enabling** the **Hidden checkbox** in the Component Attribute panel.
+
+#### Steps to use the methods for the Page Builder components
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action').
+2. Write the method/code as shown in the **Example** below.
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/check1.png">
+
+!!! Example
+    1. `this.addClassName(fields, className)`
+          * Go to Form Attribute :material-menu-right: Style Sheets :material-menu-right: add the class
+            ```
+            .abc{ // abc is the class name
+            background-color: red;
+            }
+            ```
+          * Follow the steps mentioned above, under Steps to use the methods for the Page Builder components
+          * ```
+            this.addClassName('collapse_itxbeisp', 'abc')
+            ```
+
+    2. `this.removeClassName(fields, className)`
+        ```js
+        this.removeClassName('collapse_itxbeisp', 'abc')
+        ```
+    
+    3. `this.hide(fields)`
+        ```js
+        var fields= ['collapse_itxbeisp']
+        this.hide(fields)
+        ```
+    
+    4. `this.show(fields)`
+        ```js
+        var fields= ['collapse_itxbeisp']
+        this.show(fields)
+        ```
+    
+    5. `this.getComponent('component_ID')`
+        ```js
+        var collapsename = this.getComponent('collapse_itxbeisp');
+        console.log('getComponent', collapsename);
+        ```
+
+!!! info Add multiple classes
+    Use the following syntax to add multiple classes to a component:
+    ```js
+    this.addClassName('componentID', 'class1', 'class2')
```

Comment:
This is not working. Only class1 will be added the other class2 will be ignored

Reply:
it's not working.. rechecked with Saurabh and removed this information.